### ISSUE TITLE
Parsing dont work right, time hopping...

### ISSUE THREAD

**Issue by Dexus** [simplereach/timeutils#5]
```golang
ts := timeutils.NewTime(time.Now(), timeutils.TimestampNano)
tts, _ := timeutils.ParseDateString(ts.UTC().String())
tts.String()
```
the `tts.String()` Time hopping always around, we have now 11:54 but the output hopping to 12:07, 11:50 etc. 

EDIT: Info to system: Alpine Linux (Linux 4.5.7-std-3 x86_64 ); go version go1.8.1 linux/amd64

**Comment by Dexus** [comment]
Okay my fault, is a proble of the used parser aprox

**Comment by elubow** [comment]
The issue with trying to parse millis and nanos is that you don't know if it is millis or nanos without it being explicitly stated.  Hence the reason we chose millis because it more suited the purpose at the time. Are you asking if I can add a ParseNano function?

**Comment by Dexus** [comment]
Yes, maybe it would nice to have the function. Because i use this in JSON and get only hopping dates. We have over 40k req/s so we need the exact time for problem reports.

And cause of the given TimestampNano i thought it will also parse it correct.
